components/modal/frame.js

```diff
@@ -0,0 +1,148 @@
+/**
+ * External dependencies
+ */
+import clickOutside from 'react-click-outside';
```

Comment:
Would [`withFocusOutside`](https://github.com/WordPress/gutenberg/tree/master/components/higher-order/with-focus-outside) work as a valid substitute here, acknowledging the fact that we're also containing focus, but highlighting the fact that a click outside will also have the same effect as a focus outside.

It's my hope to be able to drop `react-click-outside` altogether, since it's more often than not the wrong tool for the job.

Reply:
>Would [`withFocusOutside`](https://github.com/WordPress/gutenberg/tree/master/components/higher-order/with-focus-outside) work as a valid substitute here, acknowledging the fact that we're also containing focus, but highlighting the fact that a click outside will also have the same effect as a focus outside.
>
>It's my hope to be able to drop `react-click-outside` altogether, since it's more often than not the wrong tool for the job.

This comment was never addressed or responded.